wp-content/plugins/core/src/Integrations/Gravity_Forms/Form_Markup.php

```diff
@@ -63,38 +63,38 @@ public function add_gf_select_field_class( string $classes, $field, array $form
 		$class_icon_simple  = $this->activate_icons ? ' form-control-icon' : '';
 		$class_icon_complex = $this->activate_icons ? ' form-control-icon-complex' : '';
 
-		if ( $field['type'] === 'multiselect' || $field['inputType'] === 'multiselect' ) {
+		if ( $field['type'] === 'multiselect' ) {
 			$classes .= ' gf-multi-select';
-		} elseif ( $field['type'] === 'select' || $field['inputType'] === 'select' ) {
+		} elseif ( $field['type'] === 'select' ) {
 			$classes .= ' gf-select';
 			// Not Chosen, regular select
-			if ( ! $field['enableEnhancedUI'] ) {
+			if ( array_key_exists( 'enableEnhancedUI', $field ) && ! $field['enableEnhancedUI'] ) {
```

Comment:
```suggestion
			if ( empty( $field['enableEnhancedUI'] ) ) {
```
This one could be simplified since we're just checking for a negative value